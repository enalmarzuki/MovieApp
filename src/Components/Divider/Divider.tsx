import React from 'react';
import {StyleSheet} from 'react-native';
import {Divider as DividerRNP} from 'react-native-paper';
import {colors} from '../../Utils/colors';

const Divider = () => {
  return <DividerRNP style={styles.dividerStyle} />;
};

export default Divider;

const styles = StyleSheet.create({
  dividerStyle: {
    borderColor: colors.outlineColor,
    borderWidth: 0.2,
  },
});
