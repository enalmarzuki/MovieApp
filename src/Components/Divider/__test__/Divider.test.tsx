import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import Divider from '../Divider';
jest.useFakeTimers();

describe('should render Divider', () => {
  it('should render correctly', () => {
    const DividerSnap = renderer.create(<Divider />).toJSON();
    expect(DividerSnap).toMatchSnapshot();
  });
});
