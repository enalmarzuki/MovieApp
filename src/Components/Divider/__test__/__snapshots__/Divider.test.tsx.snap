// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Divider should render correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "rgba(0, 0, 0, 0.12)",
        "height": 0.5,
      },
      undefined,
      {
        "borderColor": "#515151",
        "borderWidth": 0.2,
      },
    ]
  }
/>
`;
